src/plugins/VectorizationTuner.cpp

```diff
@@ -165,7 +153,7 @@ void VectorizationTuner::tune(std::vector<Component>& ComponentList) {
 
 }
 
-void VectorizationTuner::iterateOwn(long long int numRepetitions,
+void VectorizationTuner::iterateOwn(unsigned int numRepetitions,
```

Comment:
change from 64bit to 32bit intended? Would `size_t` fit better here?
(This same change occurs very often in this PR)

Reply:
yes, this is intended, as the number of repetitions is not that big and everywhere else an (unsigned) int is used...